network/index.js

```diff
@@ -18,18 +18,31 @@ import { useState, useEffect } from 'react';
 
 let unsupported;
 
+
 const useNetworkStatus = initialEffectiveConnectionType => {
-  if (typeof navigator !== 'undefined' && 'connection' in navigator && 'effectiveType' in navigator.connection) {
+  if (
+    typeof navigator !== 'undefined' &&
+    'connection' in navigator &&
+    'effectiveType' in navigator.connection
+  ) {
     unsupported = false;
   } else {
     unsupported = true;
   }
 
+  const getEffectiveConnectionType = () => {
```

Comment:
@JCofman 

According to https://developer.mozilla.org/en-US/docs/Glossary/Effective_connection_type, I don't think online/offline status does not belong to effective connection type.
I'd say it could make more sense if we had:
`const { effectiveConnectionType, onlineStatus } = useNetworkStatus();`